src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -292,20 +337,79 @@ void java_bytecode_convert_classt::convert(const classt &c)
     throw 0;
   }
 
-  // now do fields
+  // Now do fields
+  for(auto overlay_class : overlay_classes)
+  {
+    for(const auto &field : overlay_class.get().fields)
+    {
+      debug()
+        << "Adding symbol from overlay class:  field '" << field.name << "'"
+        << eom;
+      convert(*class_symbol, field);
```

Comment:
So all the fields from the overlay classes are added to the original? This might benefit from a comment